at_radial_gauges/lib/radial_gauges/simple_gauge.dart

```diff
@@ -0,0 +1,236 @@
+          animationController.stop();
+        }
+
+        setState(() {});
+      });
+
+    animationController.forward();
+  }
+
+  @override
+  void dispose() {
+    animationController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    if (animationController.value !=
+        Utils.actualValueToSweepAngleRadian(
+            actualValue: widget.actualValue, maxValue: widget.maxValue)) {
+      animationController.animateTo(
+          Utils.actualValueToSweepAngleRadian(
+              actualValue: widget.actualValue, maxValue: widget.maxValue),
+          duration: Utils.getDuration(
+              isAnimate: widget.isAnimate, userMilliseconds: widget.duration));
+    }
+    return FittedBox(
+      child: Column(
+        children: [
+          widget.titlePosition == TitlePosition.top
+              ? SizedBox(
+                  height: 20,
+                  child: widget.title,
+                )
+              : const SizedBox(
+                  height: 20,
+                ),
+          SizedBox(
+            height: widget.size,
+            width: widget.size,
+            child: Padding(
+              padding: const EdgeInsets.all(8.0),
+              child: CustomPaint(
+                painter: SimpleGaugePainter(
+                  sweepAngle: animationController.value,
+                  pointerColor: widget.pointerColor,
+                ),
+                child: Center(
+                  child: ListTile(
+                    title: widget.icon ??
+                        Text(
+                          '${Utils.sweepAngleRadianToActualValue(sweepAngle: animationController.value, maxValue: widget.maxValue).toStringAsFixed(widget.decimalPlaces)} ${widget.unit ?? ''} / ${widget.maxValue.toStringAsFixed(widget.decimalPlaces)} ${widget.unit ?? ''}',
+                          style: const TextStyle(fontStyle: FontStyle.italic),
+                          textAlign: TextAlign.center,
+                        ),
+                    subtitle: widget.icon != null
+                        ? Text(
+                            '${Utils.sweepAngleRadianToActualValue(sweepAngle: animationController.value, maxValue: widget.maxValue).toStringAsFixed(widget.decimalPlaces)} ${widget.unit ?? ''}',
+                            style: const TextStyle(fontStyle: FontStyle.italic),
+                            textAlign: TextAlign.center,
+                          )
+                        : null,
+                  ),
+                ),
+              ),
+            ),
+          ),
+          const SizedBox(
+            height: 10,
+          ),
+          widget.titlePosition == TitlePosition.bottom
+              ? SizedBox(
+                  height: 30,
+                  child: widget.title,
+                )
+              : const SizedBox(
+                  height: 20,
+                )
+        ],
+      ),
+    );
+  }
+}
+
+class SimpleGaugePainter extends CustomPainter {
```

Comment:
Can we keep `SimpleGaugePainter` in a separate file?

Reply:
I kept it in the same file since that class is only being used by the SimpleGauge stateful widget. 

Given the above explanation what do you recommend we do?